app/build.gradle

```diff
@@ -91,11 +91,19 @@ android {
     targetCompatibility JavaVersion.VERSION_1_8
   }
   kotlinOptions {
+    useIR = true
```

Comment:
Is this needed? the [docs](https://kotlinlang.org/docs/whatsnew14.html#new-jvm-ir-backend) mention that it gets automatically enabled for compose